lcls_naming_tool/lcls_naming_tool.py

```diff
@@ -113,11 +112,11 @@ def increment_is_valid(nn_taxon):
 
     else:
         return True
-    
 
-def process_variable_is_valid(cspv_taxon):
-    # check control system process variable starts with a letter
-    match = re.search('^[a-zA-z]', cspv_taxon)
+
+def is_alphanumeric(taxon):
+    # check the element starts with a letter or number 
+    match = re.search('^[a-zA-Z0-9]', taxon)
```

Comment:
I just realized that strings have a built-in method `.isalnum()`, so you don't have to do any regex here at all

Reply:
I think isalum() checks if the entire string is alphanumeric. The control system PV might have underscores, dashes, etc.

Reply:
While this is true, you could also have considered `taxon[0].isalnum()`